### PR TITLE
fixed size for big files

### PR DESCRIPTION
**Pull request by vdiez**
bitwise operation is limited to 32 bit int.

I was checking the remote size for already transferred files with your getSize function but for large files (in my example a 12 GB video file) it was detecting different size.

### CODE REVIEW

**Comment by marsaud** [comment]
Seems there is much more than a fix for big file size here ... try to split your requests in single-purposed feature branches, please.

File: lib/api/src/getSize.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function (path, cb) {
     } else {
       let fileLength = 0
       for (let i = 0; i < file.EndofFile.length; i++) {
-        fileLength |= file.EndofFile[i] << (i * 8)
+        fileLength += file.EndofFile[i] * Math.pow(2, i * 8);
```
Comment (marsaud) [inline lib/api/src/getSize.js:13]: We follow https://github.com/feross/standard

Please do not add `;` at line endings :)

Comment (marsaud) [inline lib/api/src/getSize.js:13]: This is not true for the upstream code from https://github.com/bchelli/node-smb2, but for the newest content in lib/api/src (es6/es7 meant to be transpiled).

**Comment by vdiez** [comment]
sorry, disregard this, I messed up with subsequent commit. I'm creating a new pull request

**Pull request closed without merging**